deployment-examples/gcp/serverless/notification/README.md

```diff
@@ -0,0 +1,10 @@
+## Deploy as GCP Cloud Function
+
+1. For deployment to GCP Cloud Function, bundle the following files and folders into a .zip file and upload it in GCP console.
+
+- src
+- resources
+- index.js
```

Comment:
Can you rename this file for now to sth else, so people are not confused with entry file for extension/notification? Maybe to `index.googleFunction.js` and here you add the first step is to rename this file to `index.js`.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/9d19d0f9d55d80cb96e62806f3df9e1a9b181fd6